PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+### PR Title Example: [DEVHUB-1316] Create a script to refresh staging data with prod data
+
+## Jira Ticket:
+
+Link to jira ticket [DEVHUB-1111]
+
+Example: [DEVHUB-1316](https://jira.mongodb.org/browse/DEVHUB-1316)
```

Comment:
Do we need this example here? I think it is straightforward to copy and paste the link. I've found when working with the old template that I would spend a second deleting this before putting in the ticket.

Reply:
@ajhenri Yeah I agree was just being explicit but I will remove it.